docs/knowledgebase/getting-started/glossary.md

```diff
@@ -584,84 +545,64 @@ and [BABE](#blind-assignment-of-blockchain-extension-babe). In each slot, a subs
+efficient creation of incremental digests. This trie is exposed to the [runtime](#runtime) as
+[a simple key/value map](../../knowledgebase/advanced/storage) where both keys and values can be
+arbitrary byte arrays.
 
-An extensible, generic, configurable and modular system for constructing runtimes and sharing
-reusable components of them. Broadly speaking, this corresponds to the second usage paradigm of
-Substrate, higher-level than using Substrate Core, but lower-level than simply running Substrate
-Node with a custom configuration.
+## State Transition Function (STF))
 
-## STF (State Transition Function)
-
-The logic of a blockchain that determines how the state changes when any given block is executed. In
-Substrate, this is essentially equivalent to the Runtime.
+The logic of a [blockchain](#blockchain) that determines how the state changes when a
+[block](#block) is processed. In Substrate, this is effectively equivalent to the
+[runtime](#runtime).
 
 ## Storage Items
 
-Within the SRML, storage items are a means of providing type-safe persistent data for the runtime,
-achieved with a compiler macro, the `parity-codec` crate and the State API. All storage items must
-have a type for which the `parity-codec::Codec` trait is implemented, and, if they have a
-corresponding JSON Genesis Configuration entry, then also the `serde` traits.
+[FRAME](#frame) primitives that provide type-safe data persistence capabilities to the
+[runtime](#runtime). Learn more about storage items in the Knowledge Base article for
+[runtime storage](../../knowledgebase/runtime/storage).
 
 ## Substrate
 
-A framework and toolkit for building and deploying upgradable, modular and efficient blockchains.
-There are three usage paradigms in increasing levels of functionality and opinionation: Core, SRML
-and Node.
-
-### Substrate Core
-
-The lowest level of the three Substrate usage paradigms, this is a minimalist/purist blockchain
-building framework that contains essential functionality at the lowest level including consensus,
-block production, chain synchronisation, I/O for JSON-RPC, runtime, network synchronisation,
-database backend, telemetry, sandboxing, and versioning.
-
-### Substrate Execution Environment
-
-The runtime environment under which WebAssembly code runs in order to execute blocks.
+A flexible framework for building modular, efficient, and upgradeable [blockchains](#blockchain).
+Substrate is written in the [Rust](https://www.rust-lang.org/) programming language and is
+maintained by [Parity Technologies](https://www.parity.io/).
 
 ---
 
 ## Transaction
 
-A type of Extrinsic that includes a signature, and all valid instances cost the signer some amount
-of tokens when included on-chain. Because validity can be determined efficiently, transactions can
-be gossipped on the network with reasonable safety against DoS attacks, much as with Bitcoin and
-Ethereum.
+A type of [extrinsic](#extrinsic) that may include a signature.
```

Comment:
I think this is valuable: Because validity can be determined efficiently, transactions can	
be gossipped on the network with reasonable safety against DoS attacks, much as with Bitcoin and	
Ethereum.